exporter.go

```diff
@@ -422,3 +431,22 @@ func (e *Exporter) addOrReuseLayer(image imgutil.Image, layer layers.Layer, prev
 	e.Logger.Debugf("Layer '%s' SHA: %s\n", layer.ID, layer.Digest)
 	return layer.Digest, image.AddLayerWithDiffID(layer.TarPath, layer.Digest)
 }
+
+func (e *Exporter) makeBuildReport(layersDir string) (BuildReport, error) {
+	if e.PlatformAPI.Compare(api.MustParse("0.5")) < 0 { // platform API < 0.5
+		return BuildReport{}, nil
+	}
+	var out []BOMEntry
+	for _, bp := range e.Buildpacks {
+		if api.MustParse(bp.API).Compare(api.MustParse("0.5")) < 0 { // buildpack API < 0.5
+			continue
+		}
+		var bpBuildReport BuildReport
+		bpBuildTOML := filepath.Join(layersDir, bp.ID, "build.toml")
```

Comment:
```suggestion
		bpBuildTOML := filepath.Join(layersDir,  launch.EscapeID(bp.ID), "build.toml")
```
we can't use `bp.ID` directly here, we need to use the escaped version instead.